src/rnp/rnp.c

```diff
@@ -260,12 +261,19 @@ show_output(char *out, int size, const char *header)
 static int
 rnp_cmd(rnp_t *rnp, prog_t *p, char *f)
 {
-    const int cleartext = 1;
-    unsigned  maxsize;
-    char *    out;
-    char *    in;
-    int       ret;
-    int       cc;
+    const int  cleartext = 1;
+    unsigned   maxsize;
+    char *     out;
+    char *     in;
+    int        ret;
+    int        cc;
+    rnp_ctx_t *ctx;
+
+    ctx = rnp_cur_ctx_new();
```

Comment:
It is freed if another set_cur_ctx() call is made. So at max only single object will be kept alive until the application termination.